components/BB.js

```diff
@@ -64,13 +72,16 @@ export class BB extends Component {
 
   componentDidUpdate() {
     if (this.state.showDisabledCookieBanner && !this.props.cookiesDisabled) {
-      this.setState({ showDisabledCookieBanner: false });
+      this.setState({
+        showDisabledCookieBanner: false,
+        screenWidth: screen.width
+      });
     }
   }
 
   render() {
     const { t, url, store, homeUrl, printUrl } = this.props; // eslint-disable-line no-unused-vars
-
+    const onMobile = this.state.screenWidth < parseInt(globalTheme.min.sm, 10);
```

Comment:
I'm not sure if this is the best solution for switching between ProfileNeedsSelectorMobile and SelectionsEditor. See lines 56, 77, and 123. For testing the difference, you have to reload the page once you've resized your screen.  I'm under the assumption this won't be a problem for users in the app because they won't be changing their screen size. I am open to suggestions.

Reply:
@smcmurtry re: collapsable on mobile